semester-two/mcsl209/03-linked-list/find-common-elements.c

```diff
@@ -64,6 +64,8 @@ struct Node* createLinkedList( int size, struct Node *firstNode ) {
 
         newNode = createANode( data, NULL );
         temp->nextPtr = newNode;
+        // temp is assigned the address of newNode
+        // NOT same as temp->data = newNode->data; temp->nextPtr = newNode->nextPtr;
```

Comment:
@sumayyahc1  Can we please use the multi-line code for this and start the first letter as Capital case.

Reply:
Done.